cl/simple_pages/templates/help/alert_help.html

```diff
@@ -262,6 +263,22 @@ <h3>Enabling Real Time Alerts</h3>
        class="btn btn-danger"><i class="fa fa-heart-o"></i> Join Free.law</a>
   </p>
 
+  <h3 id="recap-search-alerts-limitations">Limitations of Real-Time RECAP Search Alerts</h3>
+  <p> RECAP Search Alerts can be triggered by case-level fields, filing-level fields, or a combination of both. This flexibility introduces complexity, which leads to certain limitations when triggering real-time alerts. For example, suppose you have a real-time alert based on a query like:</p>
+  <p><code>caseName:"Lorem vs Dolor" AND description:"Notice of Removal"</code></p>
+  <p>Now imagine a case previously named <strong>"Ipsum vs Dolor"</strong> contains a filing with the description <strong>"Notice of Removal"</strong>. If the case name is later updated to <strong>"Lorem vs Dolor"</strong>, this case would now match the alert query. </p>
+  <p>However, because the field that causes the match (caseName) is at the case level and not tied directly to the individual filing, the system cannot detect that a relevant filing now qualifies for the alert. As a result, the alert will not be triggered in real time.
```

Comment:
```suggestion
  <p>However, because the field that causes the match (<code>caseName</code>) is at the case level and not tied directly to the individual filing, CourtListener cannot detect that a relevant filing now qualifies for the alert. As a result, the alert will not be triggered in real time.
```